app/tests/api/routes/test_LearnRequest.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.testclient import TestClient
+
+
+def test_LearnRequest1(client: TestClient):
+    response = client.post(
+        "/learn",
+        json={
+            "text": "dog",
+        },
+    )
+    assert response.status_code == 204
+
+
+def test_LearnRequest2(client: TestClient):
```

Comment:
Please rename functions with more descriptive ones. Collaborators should be able to understand what scenario the tests cover by looking at their names

Reply:
Ah I see, that's how we describe it better! I will go and make changes
Thank you!